processing/src/main/java/io/druid/segment/DimensionHandlerUtil.java

```diff
@@ -30,12 +30,21 @@ private DimensionHandlerUtil() {}
 
   public static DimensionHandler getHandlerFromCapabilities(String dimensionName, ColumnCapabilities capabilities)
   {
+    return getHandlerFromCapabilities(dimensionName, capabilities, null);
+  }
+
+  public static DimensionHandler getHandlerFromCapabilities(
```

Comment:
This method isn't called very often, so I think it's better to have just one signature (the 3-arg signature) so call sites have to be explicit about their `multiValueHandling`. IMO this will make bugs less likely.